whois-api/src/main/java/net/ripe/db/whois/api/apiKey/BearerTokenExtractor.java

```diff
@@ -46,25 +49,25 @@ public OAuthSession extractBearerToken(final HttpServletRequest request, final S
 
     private OAuthSession getOAuthSession(final String bearerToken, final String apiKeyId) {
         if(StringUtils.isEmpty(bearerToken)) {
-            return new OAuthSession(apiKeyId);
+            return new OAuthSession(apiKeyId, this.environment);
         }
 
         try {
             final SignedJWT signedJWT = SignedJWT.parse(StringUtils.substringAfter(bearerToken, "Bearer "));
 
             if(!verifyJWTSignature(signedJWT)) {
               LOGGER.debug("JWT signature verification failed for {}", apiKeyId);
-              return new OAuthSession(apiKeyId);
+              return new OAuthSession(apiKeyId, this.environment);
```

Comment:
If API key is invalid no need to add environment. Also make sure the audit logs also have this variable.

Reply:
Done